### PR TITLE
Update aptos-move/framework/README.md

### PR DESCRIPTION
**Pull request by alinush**
### Description

Updated `aptos-move/framework/README.md` with instructions on how to run the Move unit tests.

### CODE REVIEW

File: aptos-move/framework/README.md [hunk 1]
```diff
@@ -34,6 +34,40 @@ aptos move document --help
 
 The documentation process is also integrated into the framework building process and will be automatically triggered like other derived artifacts, via `cached-packages` or explicit release building.
 
+## Running Move tests
+
+To test our Move code while developing the Aptos Framework, run `cargo test` inside this directory:
+
+```
+cargo test
```
Comment (gregnazario) [inline aptos-move/framework/README.md:42]: `cargo test -p aptos-framework` works anywhere to run the tests as well

File: aptos-move/framework/README.md [hunk 2]
```diff
@@ -34,6 +34,40 @@ aptos move document --help
...
+
+(See tests in `tests/move_unit_test.rs` to figure out what filter to use; e.g., to run the tests in `aptos_framework` you must confusingly filter by `move_framework`.)
+
+Sometimes, Rust likes to run out of stack memory for no reason. In that case, try setting the following in your environment (e.g., in `~/.profile`)
+
+```
+export RUST_MIN_STACK=4297152
+```
+
+Alternatively, compiling in release mode seems to also address the stack issue:
```
Comment (gregnazario) [inline aptos-move/framework/README.md:65]: Let's work on the wording here a bit.


```suggestion
Sometimes, Rust runs out of stack memory in dev build mode.  You can address this by either:
1. Adjusting the stack size

export RUST_MIN_STACK=4297152

2. Compiling in release mode

cargo test --release -- --skip prover

```

File: aptos-move/framework/README.md [hunk 3]
```diff
@@ -34,6 +34,40 @@ aptos move document --help
...
+cargo test -- --skip prover
+```
+
+To filter and run only the tests in specific packages (e.g., `aptos_stdlib`), run:
+
+```
+cargo test -- aptos_stdlib --skip prover
+```
+
+(See tests in `tests/move_unit_test.rs` to figure out what filter to use; e.g., to run the tests in `aptos_framework` you must confusingly filter by `move_framework`.)
```
Comment (gregnazario) [inline aptos-move/framework/README.md:57]: Another one we should reword

```suggestion
(See tests in `tests/move_unit_test.rs` to determine which filter to use; e.g., to run the tests in `aptos_framework` you must filter by `move_framework`.)
```

File: aptos-move/framework/README.md (hunk 1 above)
Comment (alinush) [reply aptos-move/framework/README.md:42]: Fixed.

**Review by gregnazario — APPROVED** [review]

**Review by JoshLind — APPROVED** [review]

**Pull request merged**